.eslintrc.json

```diff
@@ -48,7 +48,7 @@
           "max-len": [
             "error",
             {
-              "code": 120
+              "code": 100
```

Comment:
I am fine with either but if we change to 100 we should also change prettier config in package.json.